src/geometry/PolySet.h

```diff
@@ -37,6 +37,7 @@ class PolySet : public Geometry
 
   bool is_convex() const;
   boost::tribool convexValue() const { return this->convex; }
+  bool isTriangular = false;
```

Comment:
Should probably be a private member with a const accessor

Reply:
I see what you mean. Let's keep this for now. This is really just another argument for making triangle meshes a separate type.